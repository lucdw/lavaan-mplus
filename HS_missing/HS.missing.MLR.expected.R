mplus.out <- "HS.missing.MLR.expected.out" # needed for batch-execution
library(lavaan)

Data <- read.table("HS.missing.raw", na.strings = "-999999", 
col.names = c("x1", "x2", "x3", "x4", "x5", "x6", "x7", "x8", "x9"))

model <- '
visual  =~ x1 + x2 + x3
textual =~ x4 + x5 + x6
speed   =~ x7 + x8 + x9
'
fit <-  sem (model, data = Data
    , estimator  = "MLR"
    , information  = "expected"
    , missing  = "ml"
    )
summary(fit)  # summary(...): removed if executed in batch
