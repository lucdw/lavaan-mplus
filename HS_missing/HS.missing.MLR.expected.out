Mplus VERSION 8.2
MUTHEN & MUTHEN
01/20/2024   8:47 PM

INPUT INSTRUCTIONS



      title:
                      [This syntax is autogenerated]
                      CFA Mplus syntax
      data:
                      file is HS.missing.raw;
      variable:
                      names are
                      x1  x2  x3  x4  x5  x6
                      x7  x8  x9  ;
                      missing are all (-999999);
      analysis:
      !               model=nomeanstructure;
                      estimator=MLR;
                      information=expected;
                      convergence=0.00000001;

      model:
                      visual by x1;
                      visual by x2;
                      visual by x3;
                      textual by x4;
                      textual by x5;
                      textual by x6;
                      speed by x7;
                      speed by x8;
                      speed by x9;
      Output: standardized;









INPUT READING TERMINATED NORMALLY




[This syntax is autogenerated]
CFA Mplus syntax

SUMMARY OF ANALYSIS

Number of groups                                                 1
Number of observations                                         301

Number of dependent variables                                    9
Number of independent variables                                  0
Number of continuous latent variables                            3

Observed dependent variables

  Continuous
   X1          X2          X3          X4          X5          X6
   X7          X8          X9

Continuous latent variables
   VISUAL      TEXTUAL     SPEED


Estimator                                                      MLR
Information matrix                                        EXPECTED
Maximum number of iterations                                  1000
Convergence criterion                                    0.100D-07
Maximum number of steepest descent iterations                   20
Maximum number of iterations for H1                           2000
Convergence criterion for H1                             0.100D-03

Input data file(s)
  HS.missing.raw

Input data format  FREE


SUMMARY OF DATA

     Number of missing data patterns            36


COVARIANCE COVERAGE OF DATA

Minimum covariance coverage value   0.100


     PROPORTION OF DATA PRESENT


           Covariance Coverage
              X1            X2            X3            X4            X5
              ________      ________      ________      ________      ________
 X1             0.934
 X2             0.867         0.934
 X3             0.867         0.870         0.934
 X4             0.870         0.880         0.867         0.934
 X5             0.870         0.874         0.870         0.874         0.934
 X6             0.867         0.880         0.874         0.867         0.870
 X7             0.870         0.874         0.874         0.867         0.867
 X8             0.870         0.870         0.870         0.874         0.874
 X9             0.870         0.870         0.867         0.874         0.870


           Covariance Coverage
              X6            X7            X8            X9
              ________      ________      ________      ________
 X6             0.934
 X7             0.870         0.934
 X8             0.870         0.870         0.934
 X9             0.867         0.870         0.870         0.934



UNIVARIATE SAMPLE STATISTICS


     UNIVARIATE HIGHER-ORDER MOMENT DESCRIPTIVE STATISTICS

         Variable/         Mean/     Skewness/   Minimum/ % with                Percentiles
        Sample Size      Variance    Kurtosis    Maximum  Min/Max      20%/60%    40%/80%    Median

     X1                    4.934      -0.244       0.667    0.36%       4.000      4.667      5.000
             281.000       1.407       0.279       8.500    0.36%       5.333      6.000
     X2                    6.093       0.480       2.250    0.36%       5.250      5.750      6.000
             281.000       1.379       0.338       9.250    1.78%       6.250      7.000
     X3                    2.237       0.386       0.250    0.36%       1.125      1.750      2.125
             281.000       1.274      -0.878       4.500    1.78%       2.375      3.375
     X4                    3.056       0.347       0.000    0.36%       2.000      2.667      3.000
             281.000       1.360       0.077       6.333    0.36%       3.333      4.000
     X5                    4.320      -0.330       1.000    0.71%       3.000      4.000      4.500
             281.000       1.624      -0.488       7.000    0.36%       4.750      5.500
     X6                    2.161       0.887       0.143    0.71%       1.286      1.857      2.000
             281.000       1.159       1.042       6.143    0.36%       2.286      2.857
     X7                    4.206       0.272       1.304    0.36%       3.304      3.826      4.087
             281.000       1.174      -0.253       7.435    0.36%       4.391      5.130
     X8                    5.511       0.530       3.050    0.36%       4.700      5.250      5.500
             281.000       1.024       1.305      10.000    0.36%       5.700      6.250
     X9                    5.359       0.255       2.778    0.36%       4.472      5.056      5.389
             281.000       1.026       0.372       9.250    0.36%       5.556      6.194


THE MODEL ESTIMATION TERMINATED NORMALLY



MODEL FIT INFORMATION

Number of Free Parameters                       30

Loglikelihood

          H0 Value                       -3507.165
          H0 Scaling Correction Factor      1.0397
            for MLR
          H1 Value                       -3465.850
          H1 Scaling Correction Factor      1.0400
            for MLR

Information Criteria

          Akaike (AIC)                    7074.330
          Bayesian (BIC)                  7185.544
          Sample-Size Adjusted BIC        7090.401
            (n* = (n + 2) / 24)

Chi-Square Test of Model Fit

          Value                             79.425*
          Degrees of Freedom                    24
          P-Value                           0.0000
          Scaling Correction Factor         1.0404
            for MLR

*   The chi-square value for MLM, MLMV, MLR, ULSMV, WLSM and WLSMV cannot be used
    for chi-square difference testing in the regular way.  MLM, MLR and WLSM
    chi-square difference testing is described on the Mplus website.  MLMV, WLSMV,
    and ULSMV difference testing is done using the DIFFTEST option.

RMSEA (Root Mean Square Error Of Approximation)

          Estimate                           0.088
          90 Percent C.I.                    0.067  0.109
          Probability RMSEA <= .05           0.002

CFI/TLI

          CFI                                0.927
          TLI                                0.890

Chi-Square Test of Model Fit for the Baseline Model

          Value                            794.804
          Degrees of Freedom                    36
          P-Value                           0.0000

SRMR (Standardized Root Mean Square Residual)

          Value                              0.061



MODEL RESULTS

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 VISUAL   BY
    X1                 1.000      0.000    999.000    999.000
    X2                 0.566      0.106      5.355      0.000
    X3                 0.774      0.122      6.351      0.000

 TEXTUAL  BY
    X4                 1.000      0.000    999.000    999.000
    X5                 1.087      0.068     16.012      0.000
    X6                 0.905      0.064     14.228      0.000

 SPEED    BY
    X7                 1.000      0.000    999.000    999.000
    X8                 1.210      0.168      7.214      0.000
    X9                 1.246      0.172      7.229      0.000

 TEXTUAL  WITH
    VISUAL             0.419      0.085      4.908      0.000

 SPEED    WITH
    VISUAL             0.273      0.057      4.802      0.000
    TEXTUAL            0.158      0.054      2.946      0.003

 Intercepts
    X1                 4.931      0.070     70.577      0.000
    X2                 6.105      0.070     87.573      0.000
    X3                 2.239      0.067     33.437      0.000
    X4                 3.074      0.068     45.059      0.000
    X5                 4.328      0.075     57.764      0.000
    X6                 2.166      0.063     34.336      0.000
    X7                 4.210      0.064     65.415      0.000
    X8                 5.508      0.060     92.448      0.000
    X9                 5.352      0.060     89.806      0.000

 Variances
    VISUAL             0.784      0.169      4.642      0.000
    TEXTUAL            0.996      0.122      8.136      0.000
    SPEED              0.329      0.080      4.117      0.000

 Residual Variances
    X1                 0.617      0.138      4.488      0.000
    X2                 1.126      0.107     10.562      0.000
    X3                 0.810      0.089      9.066      0.000
    X4                 0.366      0.053      6.849      0.000
    X5                 0.465      0.058      7.950      0.000
    X6                 0.348      0.048      7.212      0.000
    X7                 0.850      0.081     10.499      0.000
    X8                 0.536      0.076      7.030      0.000
    X9                 0.508      0.076      6.709      0.000


STANDARDIZED MODEL RESULTS


STDYX Standardization

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 VISUAL   BY
    X1                 0.748      0.066     11.301      0.000
    X2                 0.427      0.056      7.636      0.000
    X3                 0.606      0.055     11.015      0.000

 TEXTUAL  BY
    X4                 0.855      0.025     33.536      0.000
    X5                 0.847      0.022     38.180      0.000
    X6                 0.837      0.024     34.425      0.000

 SPEED    BY
    X7                 0.528      0.055      9.579      0.000
    X8                 0.688      0.047     14.516      0.000
    X9                 0.708      0.048     14.876      0.000

 TEXTUAL  WITH
    VISUAL             0.475      0.074      6.376      0.000

 SPEED    WITH
    VISUAL             0.538      0.073      7.403      0.000
    TEXTUAL            0.277      0.077      3.581      0.000

 Intercepts
    X1                 4.166      0.205     20.302      0.000
    X2                 5.202      0.226     23.040      0.000
    X3                 1.978      0.069     28.770      0.000
    X4                 2.634      0.114     23.079      0.000
    X5                 3.378      0.150     22.571      0.000
    X6                 2.008      0.089     22.531      0.000
    X7                 3.878      0.152     25.579      0.000
    X8                 5.462      0.286     19.123      0.000
    X9                 5.304      0.242     21.918      0.000

 Variances
    VISUAL             1.000      0.000    999.000    999.000
    TEXTUAL            1.000      0.000    999.000    999.000
    SPEED              1.000      0.000    999.000    999.000

 Residual Variances
    X1                 0.441      0.099      4.449      0.000
    X2                 0.818      0.048     17.135      0.000
    X3                 0.633      0.067      9.499      0.000
    X4                 0.269      0.044      6.171      0.000
    X5                 0.283      0.038      7.541      0.000
    X6                 0.299      0.041      7.355      0.000
    X7                 0.721      0.058     12.386      0.000
    X8                 0.527      0.065      8.085      0.000
    X9                 0.499      0.067      7.398      0.000


STDY Standardization

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 VISUAL   BY
    X1                 0.748      0.066     11.301      0.000
    X2                 0.427      0.056      7.636      0.000
    X3                 0.606      0.055     11.015      0.000

 TEXTUAL  BY
    X4                 0.855      0.025     33.536      0.000
    X5                 0.847      0.022     38.180      0.000
    X6                 0.837      0.024     34.425      0.000

 SPEED    BY
    X7                 0.528      0.055      9.579      0.000
    X8                 0.688      0.047     14.516      0.000
    X9                 0.708      0.048     14.876      0.000

 TEXTUAL  WITH
    VISUAL             0.475      0.074      6.376      0.000

 SPEED    WITH
    VISUAL             0.538      0.073      7.403      0.000
    TEXTUAL            0.277      0.077      3.581      0.000

 Intercepts
    X1                 4.166      0.205     20.302      0.000
    X2                 5.202      0.226     23.040      0.000
    X3                 1.978      0.069     28.770      0.000
    X4                 2.634      0.114     23.079      0.000
    X5                 3.378      0.150     22.571      0.000
    X6                 2.008      0.089     22.531      0.000
    X7                 3.878      0.152     25.579      0.000
    X8                 5.462      0.286     19.123      0.000
    X9                 5.304      0.242     21.918      0.000

 Variances
    VISUAL             1.000      0.000    999.000    999.000
    TEXTUAL            1.000      0.000    999.000    999.000
    SPEED              1.000      0.000    999.000    999.000

 Residual Variances
    X1                 0.441      0.099      4.449      0.000
    X2                 0.818      0.048     17.135      0.000
    X3                 0.633      0.067      9.499      0.000
    X4                 0.269      0.044      6.171      0.000
    X5                 0.283      0.038      7.541      0.000
    X6                 0.299      0.041      7.355      0.000
    X7                 0.721      0.058     12.386      0.000
    X8                 0.527      0.065      8.085      0.000
    X9                 0.499      0.067      7.398      0.000


STD Standardization

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 VISUAL   BY
    X1                 0.885      0.095      9.283      0.000
    X2                 0.501      0.075      6.703      0.000
    X3                 0.685      0.070      9.779      0.000

 TEXTUAL  BY
    X4                 0.998      0.061     16.271      0.000
    X5                 1.085      0.056     19.287      0.000
    X6                 0.903      0.061     14.883      0.000

 SPEED    BY
    X7                 0.573      0.070      8.234      0.000
    X8                 0.694      0.068     10.184      0.000
    X9                 0.714      0.063     11.326      0.000

 TEXTUAL  WITH
    VISUAL             0.475      0.074      6.376      0.000

 SPEED    WITH
    VISUAL             0.538      0.073      7.403      0.000
    TEXTUAL            0.277      0.077      3.581      0.000

 Intercepts
    X1                 4.931      0.070     70.577      0.000
    X2                 6.105      0.070     87.573      0.000
    X3                 2.239      0.067     33.437      0.000
    X4                 3.074      0.068     45.059      0.000
    X5                 4.328      0.075     57.764      0.000
    X6                 2.166      0.063     34.336      0.000
    X7                 4.210      0.064     65.415      0.000
    X8                 5.508      0.060     92.448      0.000
    X9                 5.352      0.060     89.806      0.000

 Variances
    VISUAL             1.000      0.000    999.000    999.000
    TEXTUAL            1.000      0.000    999.000    999.000
    SPEED              1.000      0.000    999.000    999.000

 Residual Variances
    X1                 0.617      0.138      4.488      0.000
    X2                 1.126      0.107     10.562      0.000
    X3                 0.810      0.089      9.066      0.000
    X4                 0.366      0.053      6.849      0.000
    X5                 0.465      0.058      7.950      0.000
    X6                 0.348      0.048      7.212      0.000
    X7                 0.850      0.081     10.499      0.000
    X8                 0.536      0.076      7.030      0.000
    X9                 0.508      0.076      6.709      0.000


R-SQUARE

    Observed                                        Two-Tailed
    Variable        Estimate       S.E.  Est./S.E.    P-Value

    X1                 0.559      0.099      5.651      0.000
    X2                 0.182      0.048      3.818      0.000
    X3                 0.367      0.067      5.507      0.000
    X4                 0.731      0.044     16.768      0.000
    X5                 0.717      0.038     19.090      0.000
    X6                 0.701      0.041     17.213      0.000
    X7                 0.279      0.058      4.790      0.000
    X8                 0.473      0.065      7.258      0.000
    X9                 0.501      0.067      7.438      0.000


QUALITY OF NUMERICAL RESULTS

     Condition Number for the Information Matrix              0.389E-02
       (ratio of smallest to largest eigenvalue)


     Beginning Time:  20:47:34
        Ending Time:  20:47:34
       Elapsed Time:  00:00:00



MUTHEN & MUTHEN
3463 Stoner Ave.
Los Angeles, CA  90066

Tel: (310) 391-9971
Fax: (310) 391-8971
Web: www.StatModel.com
Support: Support@StatModel.com

Copyright (c) 1998-2018 Muthen & Muthen
